rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+reorder_imports = true
```

Comment:
No - I removed the clippy toml since clippy was passing there without any exclusions being required :)